### ISSUE TITLE
PDF download of invoices

### ISSUE THREAD

**Issue by pwichmann** [XeroAPI/xero-node#120]
I was already discombobulated after I learned that there is no way to send invoices per email via the API. (Are you sure this is not an absolute core functionality you should offer these days?)

Is there a way to at least download the branded invoice as a PDF via the API so that I can use a mail service to send the invoice?

**Comment by securityvoid** [comment]
I have a pull request that adds this functionality. #119 

By default it is not currently supported by the NodeJS API.

**Comment by securityvoid** [comment]
You might be best to just do a:

``` npm install --save git+https://github.com/securityvoid/xero-node.git  ```

It doesn't appear that GitHub issues are actively monitored by Xero. Its at least been 9 days since I opened tickets for a number of issues I've found and no one has even commented/looked at them yet.

Most active projects get a response within 48hrs.

Once you have that version it is:
```
var invoice =  xero.getInvoice: function(invoiceId, modifiedAfter, where, "application/pdf");  
var content = invoice.PdfContentRaw;
var filename = invoice.FileName;

```

e.g.
https://github.com/XeroAPI/xero-node/compare/master...securityvoid:master

**Comment by pwichmann** [comment]
Thank you @securityvoid !

@XeroAPI: Come on, guys, get your act together! This is embarrassing. If I had time for another startup, I would work on a better billing/accounting provider.

**Comment by securityvoid** [comment]
FYI, it is in the regular "API", its just not mentioned where you'd expect:
https://developer.xero.com/documentation/api/requests-and-responses

``` Individual Invoices and Credit Notes can also be returned in PDF format by setting the “Accept” value in the http header to “application/pdf”. ```

**Comment by philals** [comment]
Hi @pwichmann. Thanks for sending PR and Issues. We will try hard to get them processed next week.

**Comment by pwichmann** [comment]
Hi @philals, any progress on xero-node?

**Comment by philals** [comment]
Hi @pwichmann please see discussion at #119

**Comment by philals** [comment]
Hi. Sorry for the delay. Sid managed to get this working so have merged and release under 2.16.0

Thanks for the input.
